src/account.ts

```diff
@@ -95,10 +99,19 @@ export class Erc20Coin extends AccountCoinToken {
  * the token code and the issuer account in the form: (t)xlm:<token>-<issuer>
  */
 export class StellarCoin extends AccountCoinToken {
-  constructor(options: AccountConstructorOptions) {
+  public domain: string;
+
+  constructor(options: StellarCoinConstructorOptions) {
     super({
       ...options,
     });
+
+    const domainPattern = /^(?:[a-z0-9](?:[a-z0-9-]{0,61}[a-z0-9])?\.)+[a-z0-9][a-z0-9-]{0,61}[a-z0-9]$/;
```

Comment:
What is this supposed to match? Any valid domain name? Looks a bit wacky, but not necessarily wrong.

Reply:
yes, I found it and tested it here: https://regexr.com/3au3g